src/report/generate_https_scan_report.py

```diff
@@ -0,0 +1,926 @@
+#!/usr/bin/env python3
+
+"""Create Cyber Hygiene HTTPS Report PDF.
+
+Usage:
+  generate_https_scan_report [options] "AGENCY"
```

Comment:
Would it be too much trouble to update this like below? I know it would require updating all references, but I'm really not a fan because it requires accessing the element as `['"AGENCY"']` which is both awkward and serves no benefit.

```suggestion
  generate_https_scan_report [options] AGENCY
```